.gitattributes

```diff
@@ -1,2 +1,5 @@
 * text=auto
 *.js text eol=lf
+/.github/ export-ignore
+/yarn.lock export-ignore
+/fievel-mousekewitz48.gif export-ignore
```

Comment:
I would make it more general to prevent future GIFs from being accidentally included:

```
*.gif export-ignore
```